modules/tensor_mechanics/src/materials/ADGeneralizedRadialReturnStressUpdate.C

```diff
@@ -55,8 +59,15 @@ ADGeneralizedRadialReturnStressUpdate::ADGeneralizedRadialReturnStressUpdate(
         _base_name + getParam<std::string>("inelastic_strain_rate_name"))),
     _max_inelastic_increment(getParam<Real>("max_inelastic_increment")),
     _max_integration_error(getParam<Real>("max_integration_error")),
-    _max_integration_error_time_step(std::numeric_limits<Real>::max())
+    _max_integration_error_time_step(std::numeric_limits<Real>::max()),
+    _angle(isParamValid("rotation_angles") ? getParam<RealVectorValue>("rotation_angles")
+                                           : RealVectorValue(0.0, 0.0, 0.0)),
+    _transformation_tensor(6),
+    _hill_constants(6)
+
 {
+  for (unsigned int i = 0; i < _transformation_tensor.size(); ++i)
```

Comment:
`_transformation_tensor` should be a `DenseMatrix` as well. Then you can initialize it as `_transformation_tensor(6, 6)`.